src/model/transferlog.js

```diff
@@ -21,27 +21,27 @@ class TransferLog {
     })
   }
 
-  getDirection (transferId) {
+  isIncoming (transferId) {
```

Comment:
Either isIncoming is this class' concern, in which case it should strip off that field in `get()`. Or it isn't, in which case, it shouldn't have an `isIncoming` method.